l1-contracts/deploy-scripts/GatewayPreparation.s.sol

```diff
@@ -272,6 +287,87 @@ contract GatewayPreparation is Script {
         saveOutput(l2TxHash);
     }
 
+    /// @dev Calling this function requires private key to the admin of the chain
```

Comment:
Unlike other scripts, it is also assumed that the rpc URL is gateway URL, we should add it here in the comments also

Reply:
not anymore, I do it via l1->L2 because of the chainAdmin